templates/desktop/developers.html

```diff
@@ -217,7 +217,7 @@ <h2>By developers, for developers</h2>
           <p>Ubuntu has been the perfect OS given its popularity with developers and its cloud capabilities. That&rsquo;s why it&rsquo;s preloaded on the 4th generation of our XPS 13 laptop and our new Precision M3800 Mobile Workstation.</p>
           <cite class="p-pull-quote__citation">Barton George, Director of developer programs at Dell services</cite>
         </blockquote>
-        <p><a href="https://insights.ubuntu.com/2015/04/09/designed-for-developers-dell-launches-two-new-ubuntu-based-systems/" class="p-link--external">Learn more about Dell&rsquo;s developer edition line</a></p>
+        <p><a href="/blog/2015/04/09/designed-for-developers-dell-launches-two-new-ubuntu-based-systems/" class="p-link--external">Learn more about Dell&rsquo;s developer edition line</a></p>
```

Comment:
The redirects works, but don't think the links updated in this PR need the `p-link--external` class anymore.